modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -26,20 +26,30 @@ const fadeIn = keyframes`
   }
 `;
 
-const Container = styled(Box)<StyledType>({
-  position: 'fixed',
-  top: 0,
-  left: 0,
-  width: '100vw',
-  height: '100vh',
-  background: 'rgba(0,0,0,0.65)',
-  animationName: `${fadeIn}`,
-  animationDuration: '0.3s',
-  // Allow overriding of animation in special cases
-  '.wd-no-animation &': {
-    animation: 'none',
+const Container = styled(Box)<StyledType>(
+  {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    width: '100vw',
+    height: '100vh',
+    background: 'rgba(0,0,0,0.65)',
+    animationName: `${fadeIn}`,
+    animationDuration: '0.3s',
+    // Allow overriding of animation in special cases
+    '.wd-no-animation &': {
+      animation: 'none',
+    },
   },
-});
+  ({theme}) => {
+    const {canvas: canvasTheme} = getTheme(theme);
+    return {
+      [canvasTheme.breakpoints.down('s')]: {
+        height: '100%',
```

Comment:
This solution comes from this [article](https://dev-tips.com/css/overlapping-bottom-navigation-bar-despite-100vh-in-ios-safari)